boost-gradle/src/test/groovy/BoostPackageSpringSSLTest.groovy

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+import java.io.File
+import java.io.IOException
+import java.io.BufferedReader
+import java.io.FileReader
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+import org.apache.http.conn.ssl.SSLContextBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertTrue
+
+import static org.gradle.testkit.runner.TaskOutcome.*
+
+public class BoostPackageSpringSSLTest extends AbstractBoostTest {
+
+    static File resourceDir = new File("build/resources/test/test-spring-boot-ssl")
+    static File testProjectDir = new File(integTestDir, "PackageSpringSSLTest")
+    static String buildFilename = "build.gradle"
+
+    private static String URL = "https://localhost:8443/"; 
+
+    @Before
+    public void setup() {
+        createDir(testProjectDir)
+        createTestProject(testProjectDir, resourceDir, buildFilename)
+        
+        BuildResult result = GradleRunner.create()
+            .withProjectDir(testProjectDir)
+            .withArguments("boostPackage", "boostStart")
```

Comment:
Add in the log forwarding:
```
BuildResult result = GradleRunner.create()
            .withProjectDir(testProjectDir)
            .forwardOutput()
            .withArguments("boostPackage", "boostStart", "-i", "-s")
            .build()
```

Reply:
You can add it below too if you want. Not sure if we really need it for boostStop though.